### PR TITLE
Changed internal variable name in Multimaps class for ImmutableSortedBagFactory

### LINKED ISSUE #737: Variable name in Multimaps class for ImmutableSortedBagFactory is incorrect
The name of the variable is currently `multimap` and should instead be `sortedBag` in the `Multimaps` class.

This is what currently exists in the `ImmutableMultimaps` static inner class.
```
public final ImmutableSortedBagMultimapFactory multimap = new ImmutableSortedBagMultimapFactory();

```
This should be changed as follows:

```
public final ImmutableSortedBagMultimapFactory sortedBag = new ImmutableSortedBagMultimapFactory();

```

### PR DESCRIPTION
**Pull request by gastaldi**
Fixes #737

### CODE REVIEW

**Comment by gastaldi** [comment]
Any chance to merge this soon?

**Comment by donraab** [comment]
Apologies @gastaldi I had reviewed the code and requested a change and thought I had submitted the review.  I have submitted it now.  I'd like to leave the badly named multimap variable for now and deprecate it.

File: eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ private Multimaps()
         public final ImmutableSetMultimapFactory set = new ImmutableSetMultimapFactory();
         public final ImmutableSortedSetMultimapFactory sortedSet = new ImmutableSortedSetMultimapFactory();
         public final ImmutableBagMultimapFactory bag = new ImmutableBagMultimapFactory();
-        public final ImmutableSortedBagMultimapFactory multimap = new ImmutableSortedBagMultimapFactory();
```
[the remark this answers is not in the document]
Comment (gastaldi) [reply eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java]: How about deprecating the badly named `multimap` and point to the same reference as in the new `sortedBag`

[the remark this answers is not in the document]
Comment (donraab) [reply eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java]: Sounds good to me.

**Comment by donraab** [comment]
Thank you for the contribution @gastaldi!

**Pull request merged**